amd_openvx_extensions/amd_rpp/source/Downmix.cpp

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright (c) 2019 - 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+#include "vx_ext_amd.h"
+#define NUM_OF_DIMS 5
+
+struct DownmixLocalData
+{
+    RPPCommonHandle *handle;
+    Rpp32u device_type;
+    Rpp32u nbatchSize;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    vx_int32 *samples;
+    vx_int32 *channels;
+    RpptDescPtr src_desc_ptr;
+    RpptDesc srcDesc;
+    RpptDesc dstDesc;
+    RpptDescPtr dst_desc_ptr;
+    size_t in_tensor_dims[NUM_OF_DIMS];
+    size_t out_tensor_dims[NUM_OF_DIMS];
+    vx_enum in_tensor_type;
+    vx_enum out_tensor_type;
+#if ENABLE_OPENCL
+    cl_mem cl_pSrc;
+    cl_mem cl_pDst;
+#elif ENABLE_HIP
+    void *hip_pSrc;
+    void *hip_pDst;
+#endif
+};
+
+static vx_status VX_CALLBACK refreshDownmix(vx_node node, const vx_reference *parameters, vx_uint32 num, DownmixLocalData *data)
+{
```

Comment:
Please change the alignment of branches as per google standard

Reply:
Changed the alignment